runner.js

```diff
@@ -56,9 +55,13 @@ function reportHandler (linter, reportFilePath, resolve, reject, logs) {
       console.log('stderr: ' + logs)
       return reject(err)
     } else {
-      const results = linter.parseResults(data)
-      const report = linter.generateReport(results)
-      return resolve(report)
+      try {
+        const results = linter.parseResults(data)
+        const report = linter.generateReport(results)
+        return resolve(report)
+      } catch (err) {
```

Comment:
So when this runs, does it immediately  fail and quit the scan?  Or does it scan all files, report on what failed due to syntax errors and what failed due to security issues?

Reply:
Right now it fails when there is an invalid file.
I will change this.